code/go/pkg/specerrors/constants.go

```diff
@@ -9,10 +9,13 @@ const (
 	UnassignedCode = ""
 
 	// PSR - Package Spec [General] Rule
-	CodeNonGASpecOnGAPackage = "PSR00001"
+	CodeNonGASpecOnGAPackage         = "PSR00001"
```

Comment:
I'd say this code can be removed, it looks like is not used right now

Reply:
It is used in https://github.com/elastic/package-spec/blob/d0c66e84c6e2ccdfb9ea0811d2f4b85e1c388a96/code/go/internal/validator/spec.go#L78

Reply:
Ah it's from other file out of this PR , sorry. I thought it was a new error code :+1: